### PR TITLE
Disable Embroider and CSS Minification

### PR DESCRIPTION
**Pull request by jrjohnson**
In order to fix issues with styles out of common I've disabled embroider by default placing it behind an environmental variable and then disabled css minification as well to fix a problem with the way colors were showing in some places. We'll need to investigate both of these, but for now this unlocks us.

### CODE REVIEW

**Comment by jrjohnson** [comment]
@dartajax please click around and see if anything looks weird compared to production. I've switched out some CSS options here that could have strange consequences.

**Review by dartajax — COMMENTED** [review]
This is on prod ...

image

image

**Comment by dartajax** [comment]
You may be able to disregard my comment / review above since it has nothing to do with this exact PR - our Netlify current build also has the fainter borders for entry and search fields

**Comment by dartajax** [comment]
Screen shot from PR frontend ...

image

Screen shot from prod ...

image

I'm not sure if this matters or not but since I am testing, I am reporting my findings.

**Comment by dartajax** [comment]
I was trying to delete my comment above about the "borders" in the fields but I can't because it was part of my review. Sorry about that - you can disregard that comment. If it was just a comment, I would have deleted it. Just as an ...

**Comment by dartajax** [comment]
I noticed with this PR that the hours of the day on the left of the calendar now match in format when you switch between day and week view - in prod they currently do not match - I think this is actually an improvement - so far even though I have been commenting up a storm, I have found no issues that would put this in `DO NOT MERGE` territory.

**Review by dartajax — APPROVED** [review]
I did not notice anything awry when testing against production - only format improvements.

File: ember-cli-build.js [hunk 1]
```diff
@@ -52,44 +52,52 @@ module.exports = function (defaults) {
     'ember-simple-auth': {
       useSessionSetupMethod: true, //can be removed in ESA v5.x
     },
+    minifyCSS: {
```
Comment (stopfstedt) [inline ember-cli-build.js:55]: i don't think this is necessary anymore. i took this out and ran it with the `--environment=production` flag on `ember build/serve`, and it worked just fine. yanking embroider might be enough to set things right again.

**Review by stopfstedt — CHANGES_REQUESTED** [review]

File: ember-cli-build.js (hunk 1 above)
Comment (jrjohnson) [reply ember-cli-build.js:55]: Without that we lose border color on the calendar and other random places where color just doesn't get applied. It actually matches production then as @dartajax noted above, but production is currently broken.

Comment (stopfstedt) [reply ember-cli-build.js:55]: i stand corrected :+1:

**Review by stopfstedt — APPROVED** [review]

**Pull request merged**